src/main/java/com/ait/lienzo/client/core/shape/wires/picker/ColorMapBackedPicker.java

```diff
@@ -118,10 +116,10 @@ public PickerPart findShapeAt(int x, int y)
             y = (int) Math.round(temp.getY());
         }
 
-        String color = BackingColorMapUtils.findColorAtPoint(m_imageData, x, y);
+        ImageDataPixelColor color = m_ctx.getImageDataPixelColor(x, y);
         if (color != null)
         {
-            PickerPart pickerPart = m_colorMap.get(color);
+            PickerPart pickerPart = m_colorMap.get(color.toBrowserRGB());
             if (pickerPart != null)
```

Comment:
This `If` statement is redundant, method will return `null` in both cases: if `pickerPart is null` and if `pickerPart is not null` as well.